translations/player_turns.tex/es.po

```diff
@@ -56,7 +56,7 @@ msgstr ""
 #. type: Plain text
 #: sections/player_turns.tex:3
 msgid "\\addsection{Player Turns}{\\spells/dimension_door.png}"
-msgstr ""
+msgstr "\\addsection{Turnos del jugador}{\\spells/dimension_door.png}"
```

Comment:
```suggestion
msgstr "\\addsection{Turnos del jugador}{\\spells/dimension_door.png}\n\\vspace*{\\lang_header_adjustment}"
```
I think this will save some space for the first column.